main.go

```diff
@@ -82,13 +82,7 @@ func caduceus(arguments []string) int {
 		return 1
 	}
 
-	var (
-		infoLog  = log.WithPrefix(logger, level.Key(), level.InfoValue())
-		errorLog = log.WithPrefix(logger, level.Key(), level.ErrorValue())
-		debugLog = log.WithPrefix(logger, level.Key(), level.DebugValue())
-	)
-
-	infoLog.Log("configurationFile", v.ConfigFileUsed())
+	logging.Info(logger).Log("configurationFile", v.ConfigFileUsed())
```

Comment:
I just got rid of all these `logging.Info()` in caduceus to increase performance - please use
```
log.WithPrefix(logger, level.Key(), level.InfoValue()).Log(whatever...)
``` 
or add the level.Key() stuff to the log statement itself.